pkg/eve/installer/grub_installer.cfg

```diff
@@ -53,7 +53,7 @@ install_part="$cmddevice"
 cat -s boot_uid "($install_part)/boot/.uuid"
 set_global rootfs_title_suffix "-installer"
 set_global do_extra_submenus "installer_submenus"
-set_global dom0_extra_args "getty"
+set_global dom0_extra_args "getty rootdelay=10"
```

Comment:
@rene I disagree. it forces any installer boot to be 10 seconds longer. in our case it doesn't make sense since the root is on  CDROM which is not handled by this parameter anyways

Reply:
@rucoder , no, it doesn't. The kernel doesn't wait if the rootfs is already available, which is the case for most of our installations. @OhmSpectator , I don't like too much to use `rootwait` because I do want to fail if there is a real problem with root device instead of wait indefinitely... But if we agree rootwait is better for this case, I can change the logic in the script and just wait indefinitely for the root device, regardless of any kernel command line (so we wouldn't need any additional argument)...

Reply:
> The kernel doesn't wait if the rootfs is already available,

As far as I understand, it still waits. `rootdelay` is just a fixed delay. "Wait for not more than N sec" is the behaviour of the `rootwait=` (not `rootwait`) argument.

Reply:
From the documentaion:

>         rootdelay=     [KNL] Delay (in seconds) to pause before attempting to
>                        mount the root filesystem
>           rootwait     [KNL] Wait (indefinitely) for root device to show up.
>                        Useful for devices that are detected asynchronously
>                        (e.g. USB and MMC devices).
>          rootwait=     [KNL] Maximum time (in seconds) to wait for root device
>                        to show up before attempting to mount the root
>                        filesystem.

Reply:
yes, you are right, let me clarify, we load ramdisk as the initial rootfs which is already available in memory so there will be no waiting time... you can test it, even with rootdelay=3, rootdelay=10 there is no waiting time while booting the ISO...

Reply:
Ok, I understand now how it's used by the script later... Here, we are emulating the behavior ourselves. But could you please clarify why we need to emulate the kernel's behavior with the script? Doesn't the kernel wait for some reason?

Reply:
ok, I agree it might be a bit confusing if we emulate kernel's behavior, let me simplify the PR... will post an updated version...